helm/templates/dhcp.yml

```diff
@@ -128,7 +128,7 @@ data:
         {{- range $flag := .startupFlags }}
         {{ $flag }} \
         {{- end }}
-        --dhcp_service_def_id={{ .serviceDefID }} \
+        --service_def_id={{ .serviceDefID }} \
```

Comment:
should we have this section for all the containers now?

Reply:
yeah good call, I'll add it